paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutInteractor.kt

```diff
@@ -108,6 +128,41 @@ internal class DefaultPaymentMethodVerticalLayoutInteractor(
         return paymentMethodToDisplay?.toDisplayableSavedPaymentMethod(providePaymentMethodName, paymentMethodMetadata)
     }
 
+    private fun getAvailableSavedPaymentMethodAction(
```

Comment:
Nice work on these! So much easier to understand than how we do it for tabbed mode!

Reply:
thank you!